misc/python/materialize/cli/run.py

```diff
@@ -277,6 +277,7 @@ def main() -> int:
                 # Setting the listen addresses below to 0.0.0.0 is required
                 # to allow Prometheus running in Docker (misc/prometheus)
                 # access these services to scrape metrics.
```

Comment:
Maybe adjust the comment too? Otherwise someone might in the future realize that Prometheus only needs 6878 and revert the change.

Reply:
Updated!